### ISSUE TITLE
Application not running error on parallel tests

### ISSUE THREAD

**Issue by apkatsikas** [plu/pxctest#11]
Seeing this error somewhat frequently, not sure how its cropping up...

`UI Testing Failure - Application is not running, unable to get Accessibility data. Did you call -launch?`
